server/api/controller/plan_tag.js

```diff
@@ -0,0 +1,20 @@
+const Controller = require('../controller/controller');
+const PlanTag = require('../model/plan_tag');
+const Log = require('../lib/log');
+
+
+class PlanTagController extends Controller {
+	/**
+	 * Return tags for this plan.
+	 * @param {IncomingRequest} req
+	 */
+	byPlan (req) {
+		return this.model.byPlan(req.params.plan_id).then(collection => {
+			Log.debug(this.tableName, 'Get tag list', req.params.plan_id);
```

Comment:
What fields of the tag it returns? Its ID or populate it with the tag table to get its labels?

Reply:
It returns the tag name and tag display name from the Tag table. See https://github.com/muamichali/meirim/blob/7583ef1a2cfe5292725020c9e3e41870d8323ba8/server/api/model/plan_tag.js#L37